Userland/Applications/Browser/Tab.gml

```diff
@@ -11,6 +11,26 @@
         }
     }
 
+    @GUI::Frame {
+        name: "webdriver_banner"
+        shape: "Box"
+        shadow: "Sunken"
+        thickness: 1
+        preferred_height: "shrink"
+        foreground_role: "TooltipText"
+        background_role: "Tooltip"
```

Comment:
Not entirely happy with this, but we don't have a "warning" color or similar in themes. It's a best-effort attempt at ensuring that text is legible across themes without hardcoding a color that would look bad with some themes.